playlist_souffle/delivery/aws_lambda/fetch_spotify_access_token.py

```diff
@@ -59,7 +59,7 @@ def fetch_spotify_access_token(refresh_token, client_id, client_secret):
     return access_token
 
 
-def main(event, context):
+def handler(event, context):
```

Comment:
one question - should the auth logic be part of a use case, entity, or here in delivery?

Reply:
Honestly i didnt do any reorganizing of the auth functions in this PR. Could probably do that

Reply:
👍

Reply:
Will probably leave this off to a future PR.